abstract-level.js

```diff
@@ -65,21 +65,11 @@ class AbstractLevel extends EventEmitter {
         closing: true,
         closed: true,
         write: true,
-        put: true,
-        del: true,
-        batch: true,
         clear: true
       }
     })
 
-    // Monitor event listeners
-    this.#eventMonitor = new EventMonitor(this, [
-      { name: 'write' },
-      { name: 'put', deprecated: true, alt: 'write' },
-      { name: 'del', deprecated: true, alt: 'write' },
-      { name: 'batch', deprecated: true, alt: 'write' }
-    ])
-
+    this.#eventMonitor = new EventMonitor(this, [{ name: 'write' }])
```

Comment:
Should we target removing multiple events complexity in this lib soon?

Reply:
Can you please elaborate? Not sure what you're referring to.

If you mean removing `EventMonitor` then it still does have a purpose (besides the deprecation messages). Namely that checking `if (this.#eventMonitor.write)` is faster than `if (this.listenerCount('write') > 0`.

Reply:
EventMonitor now only monitors a single event

Reply:
Simplified that.